mgmt_cli_test.py

```diff
@@ -830,13 +829,6 @@ def test_enospc_before_restore(self):
                 assert final_status == TaskStatus.ERROR, \
                     f"The restore task is supposed to fail, since node {target_node} lacks the disk space to download" \
                     f"the snapshot files"
-
```

Comment:
could be useful to at least print error mesage with log.info level for clarity in case someone once needed to verify the flow.

Reply:
What kind of message you'd like to see in log.info for such case?
I supposed the assertion message is quite enough to understand the testing scenario if someone wants to verify it.

Reply:
I meant to log `full_progress_string = restore_task.progress_string(parse_table_res=False,
                                                                        is_verify_errorless_result=True).stdout` instead of asserting it (as its prone to change)

Reply:
I got your point, but it seems to me it makes little sense.. When we analyze Manager failures, we usually look into sct python log. It contains all sctool outputs logged. `full_progress_string` can be easily found there.

Reply:
ok, looks it's good to go then